pkg/core/interops.go

```diff
@@ -112,21 +112,21 @@ var systemInterops = []interop.Function{
 	{Name: "System.Runtime.Notify", Func: runtimeNotify, Price: 1, RequiredFlags: smartcontract.AllowNotify},
 	{Name: "System.Runtime.Platform", Func: runtimePlatform, Price: 1},
 	{Name: "System.Runtime.Serialize", Func: runtimeSerialize, Price: 1},
-	{Name: "System.Storage.Delete", Func: storageDelete, Price: 100,
+	{Name: "System.Storage.Delete", Func: storageDelete, Price: StoragePrice,
 		AllowedTriggers: trigger.Application, RequiredFlags: smartcontract.AllowModifyStates},
-	{Name: "System.Storage.Find", Func: storageFind, Price: 1,
+	{Name: "System.Storage.Find", Func: storageFind, Price: 1000000,
 		AllowedTriggers: trigger.Application, RequiredFlags: smartcontract.AllowStates},
```

Comment:
Just noticed that `AllowStates` is post-preview2. We can keep it for now, it shouldn't affect any proper-behaving contracts, but it's a deviation still.

Reply:
Maybe we can 'deactivate' it before the release if need be.